api/RTCPeerConnection.json

```
@@ -2054,6 +2106,56 @@
           }
         }
       },
+      "icegatheringstatechange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/icegatheringstatechange_event",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#event-icegatheringstatechange",
+          "description": "<code>icegatheringstatechange</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "59"
+            },
+            "chrome_android": {
+              "version_added": "59"
+            },
+            "edge": {
+              "version_added": "15"
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "37"
```

```suggestion
              "version_added": "46"
```

Based on the Chromium version, unless you have other evidence for Opera.